### PR TITLE
contrib/inkscape_tools: Report invalid files and cleaner Makefile

### PR DESCRIPTION
**Pull request by xymus**

### CODE REVIEW

File: contrib/inkscape_tools/src/svg_to_png_and_nit.nit [hunk 1]
```diff
@@ -329,8 +329,10 @@ for drawing in drawings do
 	end
 	svg_file.close
 
-	assert page_width != -1
-	assert page_height != -1
+	if page_width == -1 or page_height == -1 then
+		stderr.write "Source drawing file '{drawing}' doesn't look like an SVG file\n"
```
Comment (privat) [inline contrib/inkscape_tools/src/svg_to_png_and_nit.nit]: `print_error` ?

**Comment by privat** [comment]
+1

File: contrib/inkscape_tools/src/svg_to_png_and_nit.nit (hunk 1 above)
Comment (xymus) [inline contrib/inkscape_tools/src/svg_to_png_and_nit.nit]: I preferred to stick with the style of the module.

**Comment by R4PaSs** [comment]
:shipit:

**Pull request merged**